zhaquirks/tuya/ts0601_smoke.py

```diff
@@ -82,6 +82,7 @@ def __init__(self, *args, **kwargs):
             ("_TZE200_ntcy3xu1", "TS0601"),
             ("_TZE200_vzekyi4c", "TS0601"),
             ("_TZE204_ntcy3xu1", "TS0601"),
+            ("_TZE200_rccxox8p", "TS0601"),
```

Comment:
To sort the entries:
```suggestion
            ("_TZE200_ntcy3xu1", "TS0601"),
            ("_TZE200_rccxox8p", "TS0601"),
            ("_TZE200_vzekyi4c", "TS0601"),
            ("_TZE204_ntcy3xu1", "TS0601"),
```